src/components/Search/Filter.tsx

```diff
@@ -0,0 +1,298 @@
+import React, { ReactElement, useEffect } from 'react'
+import classNames from 'classnames/bind'
+import { addExistingParamsToUrl } from './utils'
+import Button from '@shared/atoms/Button'
+import {
+  FilterByAccessOptions,
+  FilterByTypeOptions
+} from '../../@types/aquarius/SearchQuery'
+import { useRouter } from 'next/router'
+import queryString from 'query-string'
+import styles from './Filter.module.css'
+import { useFilter, Filters, Sort } from '@context/Filter'
+import Input from '@components/@shared/FormInput'
+import Accordion from '@components/@shared/Accordion'
+
+const cx = classNames.bind(styles)
+
+interface FilterStructure {
+  id: keyof Filters
+  label: string
+  type: string
+  options: {
+    label: string
+    value: string
+  }[]
+}
+
+const filterList: FilterStructure[] = [
+  {
+    id: 'serviceType',
+    label: 'Service Type',
+    type: 'filterList',
+    options: [
+      { label: 'datasets', value: FilterByTypeOptions.Data },
+      { label: 'algorithms', value: FilterByTypeOptions.Algorithm }
+    ]
+  },
+  {
+    id: 'accessType',
+    label: 'Access Type',
+    type: 'filterList',
+    options: [
+      { label: 'download', value: FilterByAccessOptions.Download },
+      { label: 'compute', value: FilterByAccessOptions.Compute }
+    ]
+  },
```

Comment:
suggestion: keeping these here makes sense imo, as we probably want these as default filters for every portal (see below)